lib/pages/home/profile_card.dart

```diff
@@ -18,16 +18,20 @@ class _ProfileCardState extends State<ProfileCard> {
   void initState() {
     super.initState();
     final authProvider = Provider.of<AuthProvider>(context, listen: false);
-    authProvider.getProfilePictureURL().then((value) => setState(() {
-          profilePictureURL = value;
-        }));
+    authProvider.getProfilePictureURL().then(
+          (value) => setState(
+            () {
+              profilePictureURL = value;
+            },
+          ),
+        );
```

Comment:
Suggestion for readability:
```suggestion
    authProvider.getProfilePictureURL().then(
          (value) => setState(() => profilePictureURL = value)
        );
```

Reply:
formatted this